src/news-update/index.md

```diff
@@ -0,0 +1,21 @@
+# What is a news update?
+
+News updates are a great way to garner your game some new or renewed attention and they are the primary means of staying in contact with your game's followers. Post news updates whenever something interesting happens with your game. Followers will be sent notifications and the news will be archived on your game page. Your news updates might even show up on the Game Jolt homepage.
+
+The best news updates deliver information that a game's followers would want to know. If you upload a new build of your game, you should take the time to make a news update. If your game just won a jam or an award, post a news update about it. If someone published a glowing review of your game, go ahead and make a news update.
```

Comment:
remove current examples "just won a jam or an award, post a news update about it. If someone published a glowing review of your game, go ahead and make a news update." and replace with these: hitting the milestone on development, announcing a new version, change logs, teasers.